exporter/zipkin/zipkin.go

```diff
@@ -50,6 +54,36 @@ func NewExporter(reporter reporter.Reporter, localEndpoint *model.Endpoint) *Exp
 	}
 }
 
+type Options struct {
+	// EndpointHostPort is the host:port combination for the Zipkin service
+	// e.g. 192.168.0.2:8080
+	EndpointHostPort string
+
+	// ReporterURI is the HTTP endpoint to which spans will be sent
+	// e.g. http://localhost:9411/api/v2/spans
+	ReporterURI string
+
+	// ReporterOptions are optional Options
+	// that modify the zipkin http Reporter.
+	ReporterOptions []ziphttp.ReporterOption
+}
+
+func NewExporterWithOptions(o *Options) (*Exporter, error) {
+	if o == nil {
+		// TODO: Perhaps fill in the defaults that OpenZipkin uses
+		o = new(Options)
+	}
+	if _, err := url.Parse(o.ReporterURI); err != nil {
+		return nil, fmt.Errorf("Failed to parse ReporterURI err: %v", err)
```

Comment:
why not accept a URL and avoid this?

Reply:
I just want to accept a string and not have to make the user create a URL on their own, plus it makes for easier usage if the user can just say
```go
exporter, err := zipkin.NewExporter(&zipkin.Options{ReporterURI:"https://example.com", EndpointHostPort:"example.com:9987"})
if err != nil {
   log.Fatalf("zip.NewExporter: %v", err)
}
```

Reply:
+1 to accept an string.